README.md

```diff
@@ -37,12 +37,21 @@ If you are using this on production, consider [buying me a coffee](https://ko-fi
 
 ## Configuration
 
-- [Custom variables](#custom-variables)
-- [Scheduled runs](#scheduled-runs)
-- [Disable check for some links](#disable-check-for-some-links)
-- [Check only modified files in a pull request](#check-only-modified-files-in-a-pull-request)
-- [Status code 429: Too many requests](#too-many-requests)
-- [GitHub links failure fix](#github-links-failure-fix)
+- [GitHub Action - Markdown link check 🔗✔️](#github-action---markdown-link-check-️)
+  - [How to use](#how-to-use)
+    - [Real-life usage samples](#real-life-usage-samples)
+  - [Configuration](#configuration)
+    - [Custom variables](#custom-variables)
+      - [Sample workflow with variables](#sample-workflow-with-variables)
+    - [Scheduled runs](#scheduled-runs)
+      - [Sample workflow with scheduled job](#sample-workflow-with-scheduled-job)
+    - [Disable check for some links](#disable-check-for-some-links)
+    - [Check only modified files in a pull request](#check-only-modified-files-in-a-pull-request)
+    - [Check multiple directories and files](#check-multiple-directories-and-files)
+    - [Too many requests](#too-many-requests)
+    - [GitHub links failure fix](#github-links-failure-fix)
+  - [Example Usage](#example-usage)
+  - [Versioning](#versioning)
```

Comment:
```suggestion
```

Since GitHub now provides the page index automatically, we can get rid of this list. However, if you want to keep it, maybe we can move it before the `Configuration` section.